modules/remotecontrol/Controller.js

```diff
@@ -209,6 +253,22 @@ export default class Controller extends RemoteControlParticipant {
     }
 
     /**
+     * Returns true if the remote control session is started.
+     * @returns {boolean}
+     */
+    isStarted() {
+        return this.controlledParticipant !== null;
+    }
+
+    /**
+     * Returns the id of the requested participant
+     * @returns {string} this.requestedParticipant
```

Comment:
it would be helpful to write if the id is just an endpoint ID (MUC nickname) or full MUC JID